### ISSUE TITLE
Content not announced with VoiceOver and some announced from screen behind 

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#269]
1) VoiceOver is not announcing the "close button". The focus on the top seems to be on the COVID Alert image in the background. Double tapping on this closes the COVID Alert. See screenshot 1.

2) After this, the focus goes to content behind the screen and it announces those items from the behind the screen. User is not able to access other items on this screen with VoiceOver. The screenshot 2 has how the focus with VoiceOver is for content in the back.

iOS 13.5.1 on iPhone 7

![8a](https://user-images.githubusercontent.com/14280062/85960325-c1f8b880-b970-11ea-8a3d-5885510e3cc3.jpg)
![8b](https://user-images.githubusercontent.com/14280062/85960326-c2914f00-b970-11ea-806f-22a4cd237202.jpg)

**Comment by JuliannaR** [comment]
Linked to this issue upstream https://github.com/CovidShield/mobile/issues/39

**Comment by JuliannaR** [comment]
https://github.com/cds-snc/covid-shield-mobile/issues/391 issue duplicated.
